sources/packages/backend/apps/api/src/route-controllers/education-program-offering/education-program-offering.controller.service.ts

```diff
@@ -163,6 +165,9 @@ export class EducationProgramOfferingControllerService {
       programId,
       institutionId,
     );
+    // Get institution location details.
```

Comment:
Either this check can be moved down line 171 or both queries can be performed in parallel..